modules/biobambam/bammerge/main.nf

```diff
@@ -0,0 +1,38 @@
+process BIOBAMBAM_BAMMERGE {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::biobambam=2.0.183" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/biobambam:2.0.183--h9f5acd7_1':
+        'quay.io/biocontainers/biobambam:2.0.183--h9f5acd7_1' }"
+
+    input:
+    tuple val(meta), path(bam)
+
+    output:
+    tuple val(meta), path("${prefix}.bam")  ,emit: bam
+    tuple val(meta), path("*.bai")          ,optional:true, emit: bam_index
+    tuple val(meta), path("*.md5")          ,optional:true, emit: checksum
+    path "versions.yml"                     ,emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    prefix = task.ext.prefix ?: "${meta.id}"
```

Comment:
```suggestion
    def prefix = task.ext.prefix ?: "${meta.id}"
```

Reply:
No can do! We need to omit the `def` so we get a global variable that we can use in the `output` section

Reply:
This is documented somewhere but I can't recall where

Reply:
Yeah, sorry did not realize that the output could not be something such as `tuple val(meta), path("*.bam"), emit: bam`

Reply:
No problem, feel free to hit merge if there's no objection